sdk/opendp/whitenoise/client/__init__.py

```diff
@@ -34,24 +52,71 @@ def submit(self, params, uri):
         return client.executerun(details)
 
 class DatasetClient(object):
-    def register(self, dataset):
+    """
+    A client for registering, reading and releasing differentially private
+    datasets using the opendp-whitenoise service
+    """
+    def __init__(self):
+        # Tag requests with this custom header for now
+        self._guid = _guid_header()
+
+    def readreleased(self, dataset_name, budget):
```

Comment:
this one does not need budget(the cost of the read is fixed /defined by the released dataset document.